xio-core/src/main/java/com/xjeffrose/xio/http/ClientChannelInitializer.java

```diff
@@ -0,0 +1,70 @@
+package com.xjeffrose.xio.http;
+
+import com.xjeffrose.xio.core.XioIdleDisconnectHandler;
+import com.xjeffrose.xio.core.XioMessageLogger;
+import com.xjeffrose.xio.pipeline.Pipelines;
+import com.xjeffrose.xio.tracing.XioTracing;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelHandler;
+import io.netty.channel.ChannelInitializer;
+import java.util.function.Supplier;
+import lombok.val;
+
+public class ClientChannelInitializer {
+  public static final String sslHandlerName = "ssl handler";
+  public static final String negotiationHandlerName = "negotiation handler";
+  public static final String codecName = "codec";
+  public static final String tracingName = "distributed tracing";
+  public static final String applicationCodecName = "application codec";
+  public static final String idleHandlerName = "idle handler";
+  public static final String messageLoggingName = "message logging";
+  public static final String requestBufferName = "request buffer";
+  public static final String appHandlerName = "app handler";
```

Comment:
is this an intelliJ recommendation? looks like overkill to me, we're not manipulating the pipeline using these labels.

Reply:
It was just for the unit testing but I can unwind that since it's mainly extra indirection